src/test/ts/part3/Part3Ex2Test.ts

```diff
@@ -112,5 +114,6 @@ describe('Part3Ex3Test', () => {
     ].join('\n'));
 
     // TODO: Write an assertion to test your changes (hint: TinyAssertions)
+    TinyAssertions.assertContentPresence(editor, {span: 1});
```

Comment:
This doesn't check the content has been underlined, so it's not a very reliable test as it could just be a span without any styles. Can you take another shot at this please?